ec/resources/rulesengine/v4policy.go

```diff
@@ -71,6 +71,18 @@ func ResourcePolicyCreate(
 	policyMap := make(map[string]interface{})
 	json.Unmarshal([]byte(policy), &policyMap)
 	policyMap["state"] = "locked"
+	if policyMap["name"] != nil {
+		log.Printf(`
+		[WARN] Please remove policy name, "%s" in the policy.
+		It will be ignored. 
+		The policy name would be auto-generated by the provider with following format.
+		tf-policy-[customer_account_id]-[platform]-[deploy_to]-[timestamp]`,
+			policyMap["name"])
+	}
+	policyMap["name"] = fmt.Sprintf("tf-policy-%s-%s-%s-%d",
```

Comment:
This can be a const

Reply:
Why do we need const here?

Reply:
@changgyu-oh `"tf-policy-%s-%s-%s-%d"` is a static string so it seemed appropriate to me. Could go either way on it I suppose.